tasks/css.js

```diff
@@ -114,6 +115,14 @@ gulp.task('css:bundle', () => {
             sourcemaps.init()
         ))
 
+        // add custom environment variables into sass
+        .pipe(sassVariables({
```

Comment:
Do we need to update the readme config section to make people aware of this?

Reply:
Good call – have added this in